### PR TITLE
NOISSUE - Use opcua server timestamp in opcua-adapter messages

### PR DESCRIPTION
**Pull request by manuio**

### CODE REVIEW

**Review by drasko — APPROVED** [review]
LGTM

File: opcua/gopcua/subscribe.go [hunk 1]
```diff
@@ -161,38 +172,38 @@ func (c client) runHandler(sub *opcuaGopcua.Subscription, cfg opcua.Config) erro
...
 
-// Publish forwards messages from OPC-UA MQTT broker to Mainflux NATS broker
-func (c client) Publish(ctx context.Context, token string, m opcua.Message) error {
+// Publish forwards messages from the OPC-UA Server to Mainflux NATS broker
+func (c client) publish(token string, m opcua.Message) error {
 	// Get route-map of the OPC-UA ServerURI
 	chanID, err := c.channelsRM.Get(m.ServerURI)
 	if err != nil {
-		return opcua.ErrNotFoundServerURI
+		return errNotFoundServerURI
```
Comment (nmarcetic) [inline opcua/gopcua/subscribe.go:180]: You are not wrapping error messages with our new [errors package](https://github.com/mainflux/mainflux/blob/master/errors/errors.go)? This is why we built it.

**Review by nmarcetic — CHANGES_REQUESTED** [review]

File: opcua/gopcua/subscribe.go (hunk 1 above)
Comment (manuio) [reply opcua/gopcua/subscribe.go:180]: Yes I am, as you can see here: https://github.com/mainflux/mainflux/blob/4912abca987302a0a6aaa188226407c795b10655/opcua/gopcua/subscribe.go#L69

I just found unnecessary to return Redis error here.

Comment (nmarcetic) [reply opcua/gopcua/subscribe.go:180]: I see. `just found unnecessary to return Redis error here.` But this is exactly why we did this wrapping, for example, to see  Redis error, like not found, error connecting, etc... whatever there is plenty of errors from Redis API.

Anyway, thanks for the clarification.

Comment (manuio) [reply opcua/gopcua/subscribe.go:180]: I understand but in this case the error will be always "not found", which is redundant with my custom error.

Comment (nmarcetic) [reply opcua/gopcua/subscribe.go:180]: Ok if you are sure that error is always "not found", then yea redundant I agree.

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**